src/node_crypto.h

```diff
@@ -515,10 +514,15 @@ class Hmac : public BaseObject {
       : BaseObject(env, wrap),
         initialised_(false) {
     MakeWeak<Hmac>(this);
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+    ctx_ = reinterpret_cast<HMAC_CTX *>(malloc(sizeof(HMAC_CTX)));
```

Comment:
This will break 1.0.2. It needs to call `HMAC_CTX_init(ctx_)` to zero-initialize everything.